tests/bson/bson-int64-002.phpt

```diff
@@ -0,0 +1,54 @@
+--TEST--
+MongoDB\BSON\Int64 wraps 64-bit integers on 32-bit platforms
+--SKIPIF--
+<?php if (4 !== PHP_INT_SIZE) { die('skip Only for 32-bit platform'); } ?>
+--FILE--
+<?php
+
+require_once __DIR__ . '/../utils/tools.php';
+
+$tests = [
+    MongoDB\BSON\toPHP(MongoDB\BSON\fromJSON('{"max64": {"$numberLong": "9223372036854775807"}}')),
+    MongoDB\BSON\toPHP(MongoDB\BSON\fromJSON('{"min64": {"$numberLong": "-9223372036854775808"}}')),
+    // Numbers within the range of 32-bit integers will not be wrapped
+    MongoDB\BSON\toPHP(MongoDB\BSON\fromJSON('{"max32": {"$numberLong": "2147483647"}}')),
```

Comment:
Same here, please add tests just out of the range as well (i.e. 2147483648 and - 2147483649)